test/e2e_node/swap_test.go

```diff
@@ -42,103 +51,275 @@ const (
 	cgroupV1SwapLimitFile = "/memory/memory.memsw.limit_in_bytes"
 	cgroupV2SwapLimitFile = "memory.swap.max"
 	cgroupV1MemLimitFile  = "/memory/memory.limit_in_bytes"
+	swapUsageCgroupFile   = "memory.swap.current"
 )
 
-var _ = SIGDescribe("Swap", framework.WithNodeConformance(), "[LinuxOnly]", func() {
+var _ = SIGDescribe("Swap", "[LinuxOnly]", func() {
 	f := framework.NewDefaultFramework("swap-test")
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelBaseline
 
-	ginkgo.DescribeTable("with configuration", func(qosClass v1.PodQOSClass, memoryRequestEqualLimit bool) {
-		ginkgo.By(fmt.Sprintf("Creating a pod of QOS class %s. memoryRequestEqualLimit: %t", qosClass, memoryRequestEqualLimit))
-		pod := getSwapTestPod(f, qosClass, memoryRequestEqualLimit)
-		pod = runPodAndWaitUntilScheduled(f, pod)
+	ginkgo.Context("[SwapConformance]", func() {
```

Comment:
what is `SwapConformance` ? we are trying to organize the tags and avoid explosion, why is this not `Feature:Swap` https://github.com/kubernetes/kubernetes/blob/55f2bc10435160619d1ece8de49a1c0f8fcdf276/test/e2e/feature/feature.go#L17

Reply:
I inherited this PR so I'm not really sure. I think that these would probably be better as Feature:Swap.

Reply:
Note that `Feature:Swap` might cause these tests to be skipped by jobs which have a wildcard skip for `[Feature:.*]`. This will have to be checked.